docs/containers-auth.json.5.md

```diff
@@ -78,9 +81,7 @@ An entry can be removed by using a `logout` command from a container
 tool such as `podman logout` or `buildah logout`.
 
 In addition, credential helpers can be configured for specific registries and the credentials-helper
-software can be used to manage the credentials in a more secure way than depending on the base64 encoded authentication
-provided by `login`.  If the credential helpers are configured for specific registries, the base64 encoded authentication will not be used
-for operations concerning credentials of the specified registries.
+software can be used to manage the credentials in a more secure way than stroing credentials only the base64-encoded in `auth.json`.
```

Comment:
I think this is what you're going for here:
```suggestion
software can be used to manage the credentials more securely than storing only base64-encoded credentials in `auth.json`.
```

Reply:
Here, the point is “only (storing) base64-encoded”, not “only credentials”.

Would “… can be used to manage the credentials more securely than only  storing only them base64-encoded in `auth.json`.” work here?